README.md

```diff
@@ -29,6 +29,20 @@ npm install -g @datadog/datadog-ci
 yarn global add @datadog/datadog-ci
 ```
 
+### Standalone binary (**beta**)
+
+If installing nodejs in the CI is an issue, a standalone binary is provided with [releases](https://github.com/DataDog/datadog-ci/releases). Only _linux-x64_ is supported at the time. **This standalone binary is in **beta** and its stability is not guaranteed**. To install:
```

Comment:
It looks like the emphasis on `**beta**` is lost in the middle of the bold formatting, you may want to switch it in italic:
```suggestion
If installing nodejs in the CI is an issue, a standalone binary is provided with [releases](https://github.com/DataDog/datadog-ci/releases). Only _linux-x64_ is supported at the time. **This standalone binary is in *beta* and its stability is not guaranteed**. To install:
```